_transport.py

```diff
@@ -59,7 +62,13 @@ def HTTPPost(self, url, headers=None, params=None, data=None, files=None):
       return None
 
     self.LogResponseData(response)
-
+    if headers is not None:
+      if 'Authorization' in headers:
```

Comment:
This block of code can be a private function inside _transport.py so it can be used in both HTTPPost and HTTPGet.